spec/features/user_updates_review_spec.rb

```diff
@@ -28,11 +40,13 @@
     expect(page).to have_content users_review.title
     expect(page).to have_content other_review.title
 
+    users_review_elem = page.find('li', text: users_review.title)
```

Comment:
Useless assignment to variable - `users_review_elem`. Did you mean `users_review`?